Actions/AL-Go-Helper.ps1

```diff
@@ -415,6 +415,7 @@ function ReadSettings {
         "failOn"                                 = "error"
         "treatTestFailuresAsWarnings"            = $false
         "rulesetFile"                            = ""
+        "assignPremiumPlan"                      = $false
```

Comment:
Do we provide documentation for these settings?

Also, shouldn't there be anything in the release notes?

Reply:
Will add to settings documentation.